### ISSUE TITLE
Quick List - Form Group Support

### ISSUE THREAD

**Issue by qubisoftptyltd** [porscheinformatik/clarity-addons#675]
When adding a formControlName to the quick list component:

```
<clr-quick-list
    [clrAddLabel]="'Add Role'"
    [clrAllValues]="possibleRoles"
    formControlName="roles"
    class="clr-form-control clr-row"
    clrControlClasses="clr-col-lg-3 clr-col-md-4 clr-col-sm-5">
    <label class="clr-control-label clr-col-md-2 clr-col-sm-3">Roles</label>
</clr-quick-list>
```

I get the following error:

**No value accessor for form control with name**

Is this usage supported?

The solution I have come up with for now is as follows:

```
<clr-quick-list
    [clrAddLabel]="'Add Role'"
    [clrAllValues]="possibleRoles"
    (clrValuesChanged)="rolesChanged($event)"
    class="clr-form-control clr-row"
    clrControlClasses="clr-col-lg-3 clr-col-md-4 clr-col-sm-5">
    <label class="clr-control-label clr-col-md-2 clr-col-sm-3">Roles</label>
</clr-quick-list>

rolesChanged(roles: Array<ClrQuickListValue<string>>) {
    const selectedRoles = [];
    roles.forEach(selectedRole => {
      if (selectedRole.value != null) {
        selectedRoles.push(selectedRole.value);
      }
    });

    this.roleFormGroup.patchValue({ roles: selectedRoles });
}
```

Thanks

**Comment by Wallabeng** [comment]
We have form support for quick lists in our backlog, but it‘s not yet planned for a specific date. I will keep this post updated as soon as we implement it.
If you need it earlier, we always love to see contributions.

**Comment by d-m-s** [comment]
Hi @Wallabeng, do we have any update on the schedule here?

**Comment by Wallabeng** [comment]
No, currently it's not planned by us.
If someone needs it right now, they need to implement it themselves.
As our priorities are on different more urgent topics

**Comment by mrshkv** [comment]
Hi, Mobility Planner will be listening out for updates here too. Br

**Comment by Wallabeng** [comment]
Implemented with #1529
